### ISSUE TITLE
pyright 1.1.263 raises internal error: maximum call stack exceeded 

### ISSUE THREAD

**Issue by BrandonNav** [microsoft/pyright#3747]
**Describe the bug**
I am receiving a RangeError: Maximum call stack size exceeded in files in my project.
The file is a python class that represents a recursive grammar. 

**VS Code extension or command-line**
Running pyright via the commandline
This works in 1.1.242 and below and fails in any version above, including the current released versions 1.1.263

**To Reproduce**
As this is work code, I have anonymised the names of methods but retained enough to reproduce the bug
```python
class Grammar:
    @staticmethod
    def A():
        return Grammar.B
    @staticmethod
    def B():
        return Grammar.C
    @staticmethod
    def C():
        return Grammar.D
    @staticmethod
    def D():
        return Grammar.E
    @staticmethod
    def E():
        return Grammar.F
    @staticmethod
    def F():
        return Grammar.G
    @staticmethod
    def G():
        return Grammar.H
    @staticmethod
    def H():
        return Grammar.I
    @staticmethod
    def I():
        return Grammar.J
    @staticmethod
    def J():
        return Grammar.K
    @staticmethod
    def K():
        return Grammar.L
    @staticmethod
    def L():
        return Grammar.B
```

You can avoid the RangeError if you redefine `K()` to return `Grammar.B`.

It will also fail if you replace the `@staticmethod` decorator with the `@classmethod` decorator but will succeed if **no decorator is used**

**Stacktrace**

```bash
An internal error occurred while type checking file "/Users/brandon/Work/code/grammar.py": RangeError: Maximum call stack size exceeded
    at getTypeOfClassMember (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:1796:14)
    at getTypeOfMemberAccessWithBaseType (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:4680:40)
    at getTypeOfMemberAccess (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:4491:28)
    at getTypeOfExpression (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:968:30)
    at map (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:6865:65)
    at Array.map (<anonymous>)
    at getTypeOfTupleInferred (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:6865:51)
    at getTypeOfTuple (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:6784:28)
    at getTypeOfExpression (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:983:30)
    at forEach (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:16628:66)
    at Array.forEach (<anonymous>)
    at inferFunctionReturnType (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:16625:59)
    at disableSpeculativeMode (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19640:48)
    at getFunctionInferredReturnType (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19639:25)
    at getFunctionEffectiveReturnType (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19597:20)
    at inferReturnTypeIfNecessary (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19574:13)
    at getTypeOfMemberInternal (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19903:17)
    at getTypeOfClassMemberName (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:5075:36)
    at getTypeOfClassMember (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:1818:26)
    at getTypeOfMemberAccessWithBaseType (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:4680:40)
    at getTypeOfMemberAccess (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:4491:28)
    at getTypeOfExpression (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:968:30)
    at map (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:6865:65)
    at Array.map (<anonymous>)
    at getTypeOfTupleInferred (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:6865:51)
    at getTypeOfTuple (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:6784:28)
    at getTypeOfExpression (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:983:30)
    at forEach (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:16628:66)
    at Array.forEach (<anonymous>)
    at inferFunctionReturnType (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:16625:59)
    at disableSpeculativeMode (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19640:48)
    at getFunctionInferredReturnType (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19639:25)
    at getFunctionEffectiveReturnType (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19597:20)
    at inferReturnTypeIfNecessary (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19574:13)
    at getTypeOfMemberInternal (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19903:17)
    at getTypeOfClassMemberName (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:5075:36)
    at getTypeOfClassMember (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:1818:26)
    at getTypeOfMemberAccessWithBaseType (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:4680:40)
    at getTypeOfMemberAccess (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:4491:28)
    at getTypeOfExpression (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:968:30)
    at map (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:6865:65)
    at Array.map (<anonymous>)
    at getTypeOfTupleInferred (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:6865:51)
    at getTypeOfTuple (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:6784:28)
    at getTypeOfExpression (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:983:30)
    at forEach (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:16628:66)
    at Array.forEach (<anonymous>)
    at inferFunctionReturnType (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:16625:59)
    at disableSpeculativeMode (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19640:48)
    at getFunctionInferredReturnType (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19639:25)
    at getFunctionEffectiveReturnType (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19597:20)
    at inferReturnTypeIfNecessary (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19574:13)
    at getTypeOfMemberInternal (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19903:17)
    at getTypeOfClassMemberName (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:5075:36)
    at getTypeOfClassMember (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:1818:26)
    at getTypeOfMemberAccessWithBaseType (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:4680:40)
    at getTypeOfMemberAccess (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:4491:28)
    at getTypeOfExpression (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:968:30)
    at map (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:6865:65)
    at Array.map (<anonymous>)
    at getTypeOfTupleInferred (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:6865:51)
    at getTypeOfTuple (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:6784:28)
    at getTypeOfExpression (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:983:30)
    at /Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:16628:66
Error performing analysis: RangeError: Maximum call stack size exceeded
    at getTypeOfClassMember (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:1796:14)
    at getTypeOfMemberAccessWithBaseType (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:4680:40)
    at getTypeOfMemberAccess (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:4491:28)
    at getTypeOfExpression (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:968:30)
    at map (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:6865:65)
    at Array.map (<anonymous>)
    at getTypeOfTupleInferred (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:6865:51)
    at getTypeOfTuple (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:6784:28)
    at getTypeOfExpression (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:983:30)
    at forEach (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:16628:66)
    at Array.forEach (<anonymous>)
    at inferFunctionReturnType (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:16625:59)
    at disableSpeculativeMode (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19640:48)
    at getFunctionInferredReturnType (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19639:25)
    at getFunctionEffectiveReturnType (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19597:20)
    at inferReturnTypeIfNecessary (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19574:13)
    at getTypeOfMemberInternal (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19903:17)
    at getTypeOfClassMemberName (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:5075:36)
    at getTypeOfClassMember (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:1818:26)
    at getTypeOfMemberAccessWithBaseType (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:4680:40)
    at getTypeOfMemberAccess (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:4491:28)
    at getTypeOfExpression (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:968:30)
    at map (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:6865:65)
    at Array.map (<anonymous>)
    at getTypeOfTupleInferred (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:6865:51)
    at getTypeOfTuple (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:6784:28)
    at getTypeOfExpression (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:983:30)
    at forEach (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:16628:66)
    at Array.forEach (<anonymous>)
    at inferFunctionReturnType (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:16625:59)
    at disableSpeculativeMode (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19640:48)
    at getFunctionInferredReturnType (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19639:25)
    at getFunctionEffectiveReturnType (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19597:20)
    at inferReturnTypeIfNecessary (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19574:13)
    at getTypeOfMemberInternal (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19903:17)
    at getTypeOfClassMemberName (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:5075:36)
    at getTypeOfClassMember (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:1818:26)
    at getTypeOfMemberAccessWithBaseType (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:4680:40)
    at getTypeOfMemberAccess (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:4491:28)
    at getTypeOfExpression (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:968:30)
    at map (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:6865:65)
    at Array.map (<anonymous>)
    at getTypeOfTupleInferred (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:6865:51)
    at getTypeOfTuple (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:6784:28)
    at getTypeOfExpression (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:983:30)
    at forEach (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:16628:66)
    at Array.forEach (<anonymous>)
    at inferFunctionReturnType (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:16625:59)
    at disableSpeculativeMode (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19640:48)
    at getFunctionInferredReturnType (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19639:25)
    at getFunctionEffectiveReturnType (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19597:20)
    at inferReturnTypeIfNecessary (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19574:13)
    at getTypeOfMemberInternal (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:19903:17)
    at getTypeOfClassMemberName (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:5075:36)
    at getTypeOfClassMember (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:1818:26)
    at getTypeOfMemberAccessWithBaseType (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:4680:40)
    at getTypeOfMemberAccess (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:4491:28)
    at getTypeOfExpression (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:968:30)
    at map (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:6865:65)
    at Array.map (<anonymous>)
    at getTypeOfTupleInferred (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:6865:51)
    at getTypeOfTuple (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:6784:28)
    at getTypeOfExpression (/Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:983:30)
    at /Users/brandon/.npm/_npx/6f75558c21262cab/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:16628:66
```

**Additional context**
Config:
```json
{
  "include": ["work"],
  "strict": ["work"],

  "reportMissingImports": true,
  "reportUnnecessaryTypeIgnoreComment": true,

  "pythonVersion": "3.10",
  "pythonPlatform": "All",

  "typeCheckingMode": "basic"
}
```

**Comment by erictraut** [comment]
Thanks for the bug report. I've seen this crash in our telemetry, but I haven't been able to find a way to repro the problem. It's an easy bug to fix, and it will be addressed in the next release.

**Comment by BrandonNav** [comment]
Thank you for the quick fix @erictraut

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.264, which I just published. It will also be included in a future release of pylance.
